services-toolkit/how-to-guides/dynamic-provisioning-rds.hbs.md

```diff
@@ -33,23 +33,67 @@ To configure dynamic provisioning for AWS RDS service instances, you must:
 
-> **Important** The official documentation for the `Provider` includes a step to "Install Universal Crossplane".
-> You can skip this step because Crossplane is already installed as part of Tanzu Application Platform.
->
-> The documentation also assumes Crossplane is installed in the `upbound-system` namespace.
-> However, when working with Crossplane on Tanzu Application Platform, it is installed to the
-> `crossplane-system` namespace by default.
-> Ensure that you use the correct namespace when you create the `Secret` and the `ProviderConfig`
-> with credentials for the `Provider`.
+You will need to install both the top-level "family" Provider as well as the provider-aws-rds Provider.
+
+1. Create a file named `provider-family-aws.yaml` and copy in the
+   following contents:
+
+    ```yaml
+    # provider-family-aws.yaml
+
+    ---
+    # The AWS "family" Provider - manages the ProviderConfig for all other Providers in the same family.
+    # Does not have to be created explicitly, if not created explicitly it will be installed by the first Provider created
+    # in the family.
+    apiVersion: pkg.crossplane.io/v1
+    kind: Provider
+    metadata:
+      name: upbound-provider-family-aws
+    spec:
+      package: xpkg.upbound.io/upbound/provider-family-aws:v0.36.0
+      controllerConfigRef:
+        name: upbound-provider-family-aws
+    ---
+    # The AWS RDS Provider - just one of the many Providers in the AWS family.
+    # You can add as few or as many additional Providers in the same family as you wish.
+    apiVersion: pkg.crossplane.io/v1
+    kind: Provider
+    metadata:
+      name: upbound-provider-aws-rds
+    spec:
+      package: xpkg.upbound.io/upbound/provider-aws-rds:v0.36.0
+      controllerConfigRef:
+        name: upbound-provider-family-aws
+    ---
+    # The ControllerConfig applies settings to a Provider Pod.
+    # With family Providers each Provider is a unique Pod running in the cluster.
+    apiVersion: pkg.crossplane.io/v1alpha1
+    kind: ControllerConfig
+    metadata:
+      name: upbound-provider-family-aws
+    ```
+
+1. Apply the file to the Tanzu Application Platform cluster by running:
+
+    ```console
+    kubectl apply -f provider-family-aws.yaml
+    ```
+
+1. Confirm that both Providers have been installed successfully by running `kubectl get providers` and checking for INSTALLED=True and HEALTHY=TRUE.
+
+Next you must create a `ProviderConfig` for the Providers. Refer to the sections titled "Create a Kubernetes secret for AWS" and "Create a ProviderConfig" in the [Upbound documentation](https://marketplace.upbound.io/providers/upbound/provider-aws/v0.36.0/docs/quickstart).
```

Comment:
**issue (probably user error)** I'm seeing this error 
```
upbound-provider-aws-rds-031cc3ef9f6c-59c7d5df64-vz2lc provider-aws-rds E0616 19:12:03.237587       1 reflector.go:140] k8s.io/client-go@v0.26.3/tools/cache/reflector.go:169: Failed to watch *v1beta1.ProviderConfig: failed to list *v1beta1.ProviderConfig: providerconfigs.aws.upbound.io is forbidden: User "system:serviceaccount:crossplane-system:upbound-provider-aws-rds-031cc3ef9f6c" cannot list resource "providerconfigs" in API group "aws.upbound.io" at the cluster scope
``` 

Did you run into anything similar? I am running on TAP 1.5 if that matters